packages/core/src/components/ozon-content/nodes/kop.node.tsx

```diff
@@ -3,8 +3,8 @@ import { h, Fragment } from "@stencil/core";
 import { OzonContentNodeContext } from "../ozon-content-node-context.interface";
 import { OzonContentNode } from "../ozon-content-node.interface";
 
-export class OzonContentOpschriftNode implements OzonContentNode {
-  name = "Opschrift";
+export class OzonContentKopNode implements OzonContentNode {
+  name = ["Opschrift", "label", "nummer", "kop"];
```

Comment:
```suggestion
  name = ["Kop", "Label", "Nummer", "Opschrift"];
```
Het moet allemaal met een Hoofdletter, want kijk 
<img width="463" alt="Screenshot 2025-02-17 at 13 48 26" src="https://github.com/user-attachments/assets/1d7e8c9b-6fe5-4564-940c-37767b3fcaf4" />

zonder hoofdleeter worden ze niet verwerkt: dat kun je zien aan de faalback class.

Met hoofdletter resulteert het in 1 text-node:

<img width="457" alt="Screenshot 2025-02-17 at 13 49 07" src="https://github.com/user-attachments/assets/2dd2cd37-3cb3-4b4d-a6c1-2a48ae813161" />